README.md

```diff
@@ -48,6 +48,8 @@ Components specific to government-frontend are [within the application](https://
 
 When [running government-frontend](#running-the-application) locally the component guide is available at: http://government-frontend.dev.gov.uk/component-guide
 
+Configuration for the navigation links is covered [separately](docs/navigation-links.md)
```

Comment:
I would try and make it clear here that these are the navigation links displayed as part of the content pages navigation AB test. We can update the docs later if we decide to roll this out permanently.

Reply:
Good shout 👍